### ISSUE TITLE
If `EmiTags.consolodateTags` drops a tag, `EmiIngredient`s created from it are empty

### ISSUE THREAD

**Issue by quat1024** [emilyploszaj/emi#242]
Discovered this while [messing with abbie5's Packages PR adding EMI support](https://github.com/abbie5/packages/blob/d132f69deee1023b17f9eb05eba1107d1b0aac42/Xplat/src/main/java/agency/highlysuspect/packages/compat/emi/PackagesEmiPlugin.java). Link goes to a specific commit exhibiting the problem.

Packages has a tag called `packages:things_you_need_for_package_crafting` that currently has the following contents:

```json
{
	"replace": false,
	"values": [
		"minecraft:copper_ingot"
	]
}
```

It's intended to be datapackable but by default this happens to be exactly the same as the `c:copper_ingots` tag, and in fact, `EmiTags.betterTag` discards my tag in favor of the `c` one. This is all well and good except it causes `EmiTags.TAG_VALUES` to not contain an entry for it, and creating an `EmiIngredient` for `packages:things_you_need_for_package_crafting` returns an empty ingredient, breaking the emirecipe we are adding:
![image](https://github.com/emilyploszaj/emi/assets/2068021/e610345a-0d9a-4e5d-98fc-543ac97a23f6)

I should probably not make use of tags like this (should make a real recipe type and use an ingredient instead!), but it's worrisome that having two tags that just *happen* to have the same contents can cause one to break the other.

**Comment by emilyploszaj** [comment]
Yep, ran into this yesterday as well. I plan to get a resolution out today that expands the tag ignoring EMI's caches.

**Comment by emilyploszaj** [comment]
This has been fixed in 1.0.10
